raft/raft.cpp

```diff
@@ -65,12 +65,7 @@ raft::raft(
 
     this->raft_log = std::make_shared<bzn::raft_log>(this->entries_log_path(), maximum_raft_storage);
 
-    // KEP-112 Bail if the raft storage exceeds the max storage
-    if (this->raft_log->maximum_storage_exceeded())
-    {
-        LOG(error) << MSG_ERROR_MAXIMUM_STORAGE_EXCEEDED;
-        throw std::runtime_error(MSG_ERROR_MAXIMUM_STORAGE_EXCEEDED);
-    }
+    this->bail_on_exceeded_max_storage(true);
```

Comment:
Can we please call this: 

shutdown_on_exceeded_max_storage?

Reply:
Thanks! Made the change.